src/Card/CardImageCap.jsx

```diff
@@ -1,18 +1,45 @@
 import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import Skeleton from 'react-loading-skeleton';
 import CardContext from './CardContext';
 
 const CardImageCap = React.forwardRef(({
   src,
   srcAlt,
   logoSrc,
   logoAlt,
+  skeletonHeight,
+  skeletonWidth,
+  capLogoSkeleton,
+  capLogoSkeletonHeight,
+  capLogoSkeletonWidth,
   className,
 }, ref) => {
-  const { orientation } = useContext(CardContext);
+  const { orientation, isLoading } = useContext(CardContext);
   const wrapperClassName = `pgn__card-wrapper-image-cap ${orientation}`;
 
+  if (isLoading) {
+    return (
+      <div className={classNames(className, wrapperClassName)}>
+        <Skeleton
+          containerClassName="pgn__card-image-cap-loader"
+          height={skeletonHeight}
```

Comment:
If not `skeletonHeight` prop is passed, the default height for this skeleton is pretty narrow. Can we default this height to the ~140px that the image usually is?

Reply:
Actually, similarly for the `capLogoSkeletonHeight` prop; let's set a reasonable default for this skeleton as well so consumers don't always have to pass in a value.

Reply:
Ok, you are right